frontend/apps/reader/modules/readermenu.lua

```diff
@@ -237,10 +237,12 @@ function ReaderMenu:setUpdateItemTable()
     end
 
     if Device:isTouchDevice() then
+        -- Mostly concern about touch related page turn stuff, which only applies to Reader; ends up in Taps & Gestures
         self.menu_items.page_turns = require("ui/elements/page_turns")
-    else
-        -- Placed elsewhere than in Taps and gestures, with only a subset of menu items.
-        self.menu_items.page_turns_non_touch = require("ui/elements/page_turns")
+    end
+    -- While also related to page turns, this mostly concerns platform-specific features or physical keys, and applies *everywhere*
+    if Device:hasKeys() then
+        self.menu_items.platform_navigation_features = require("ui/elements/platform_navigation")
```

Comment:
(Now that you moved swipe animations out, it could be key_navigation rather than platform_navigation - which _sounds_ a bit odd to me - if it's just when Device:hasKeys().)

Reply:
Yeah, I was never a fan of the name to begin with, so something key-centric probably makes more sense ;).